src/controllers/uploads.js

```diff
@@ -228,12 +228,14 @@ function saveFileToLocal(uploadedFile, callback) {
 		},
 		function (upload, next) {
 			var storedFile = {
+				uid: uid,
 				url: nconf.get('relative_path') + upload.url,
 				path: upload.path,
 				name: uploadedFile.name,
 			};
-
-			plugins.fireHook('filter:uploadStored', { uploadedFile: uploadedFile, storedFile: storedFile }, next);
+			if (plugins.hasListeners('filter:uploadStored')) {
```

Comment:
I don't think this is necessary. If no listeners exist, it just returns what is passed in.

Reply:
Okie dokie.